src/main/scala/roguelike/extensions/Text.scala

```diff
@@ -0,0 +1,93 @@
+package roguelike.extensions
+
+import indigo._
+import roguelike.Math
+
+extension [M <: Material](t: Text[M])
```

Comment:
Rather that duplicating code for `Group` and `Text`, you _might_ be able to just use `SpatialModifiers[SceneNode]`.

Reply:
Fixed in PR #18